flink-runtime-web/web-dashboard/src/app/pages/task-manager/list/task-manager-list.component.html

```diff
@@ -28,7 +28,7 @@
   >
     <thead>
       <tr>
-        <th nzLeft="0px" [nzWidth]="'280px'">Path, ID</th>
+        <th nzLeft="0px" nzShowSort nzSortKey="id" [nzWidth]="'280px'">Path, ID</th>
```

Comment:
Can you please create a sort function like the others instead? Passing a key is not type safe.

Reply:
Thank you very much for your comment, I will modify it right away